CMakeLists.txt

```diff
@@ -1,13 +1,10 @@
 # Root CMake file for LStore
 cmake_minimum_required(VERSION 2.8)
 project(LStore C)
-if(NOT CMAKE_BUILD_TYPE)
-    set(CMAKE_BUILD_TYPE Debug)
-endif()
 
-if(NOT LSTORE_PROJECT_VERSION)
-    set(LSTORE_PROJECT_VERSION "0.5.1-unknown_commit")
-endif()
+set(CMAKE_BUILD_TYPE "Debug" CACHE STRING "Build type (Debug|Release)")
```

Comment:
Good point. There's a ReleaseWithDebugInfo build type (I think that's the
name....maybe RelWithDebInfo)

-- 
It's dark in this basement.